core/src/main/java/com/linecorp/armeria/common/RequestContextExporter.java

```diff
@@ -114,7 +123,31 @@
         }
     }
 
-    void export(Map<String, String> out, RequestContext ctx, RequestLog log) {
+    /**
+     * Returns a {@link Map} whose key is an export key set through {@code add*()} in
+     * {@link RequestContextExporterBuilder} and value is extracted from {@link RequestContext}
+     * Note that: this method returns {@code null} if current {@link RequestContext} is {@code null}.
```

Comment:
How about returning an empty `Map` instead?

Reply:
SGTM, let me address it.